### ISSUE TITLE
provide better feedback when readme has the fair software badge, but with the wrong compliance

### ISSUE THREAD

**Issue by jspaaks** [fair-software/howfairis#55]
(currently, it says you don't have the badge)

see discussion here #54

**Comment by sverhoeven** [comment]
Verified with on a GitHub README with
```markdown
 [![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B%20%20%E2%97%8F%20%20%E2%97%8F-yellow)](https://fair-software.eu)

```
and howfairis reports
```shell
howfairis https://github.com/iomega/zenodo-upload
...
Calculated compliance: ● ● ● ● ●

It seems you have not yet added the fair-software.eu badge to
your README.md. You can do so by pasting the following snippet:

[![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F-green)](https://fair-software.eu)
```

The repo has a yellow badge while howfairis reports green.

**Comment by eriktks** [comment]
solved by PR #135
